### PR TITLE
Move from centralised singleton to imported singletons

### PR DESCRIPTION
**Pull request by eleanor-heath**
https://github.com/adaptlearning/adapt_framework/issues/3009

### CODE REVIEW

File: js/mediaView.js [hunk 1]
```diff
@@ -1,7 +1,10 @@
 import Adapt from 'core/js/adapt';
+import a11y from 'core/js/a11y';
 import ComponentView from 'core/js/views/componentView';
 import 'libraries/mediaelement-and-player';
 import 'libraries/mediaelement-fullscreen-hook';
+import log from 'core/js/a11y/log';
```
Comment (cahirodoherty-learningpool) [inline js/mediaView.js:6]: typo here. Path shouldn't include a11y

Comment (cahirodoherty-learningpool) [inline js/mediaView.js:6]: Adapt.log comes from 'core/js/logging'

**Review by oliverfoster — APPROVED** [review]

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by joe-allen-89 — APPROVED** [review]
Tested with v5.19.0.

**Pull request merged**